docs/crashlytics/android-setup.md

```diff
@@ -55,7 +55,18 @@ apply plugin: 'com.google.firebase.crashlytics'
 // ..
 ```
 
-## 4. (Optional) Enable Crashlytics NDK reporting
+## 4. Add line for implementation
+
+Inside the `android/build.gradle` file, in the dependencies add the following implementations:
+
+```
+implementation 'com.google.firebase:firebase-crashlytics'
+implementation 'com.google.firebase:firebase-analytics'
```

Comment:
These dependencies are handled by the gradle file our module includes.
Analytics is optional, you include it if you like, it will work without it.
Crashlytics is included.

https://github.com/invertase/react-native-firebase/blob/b8f80a41e864c2c8d97602df625abdd2ee6bc347/packages/crashlytics/android/build.gradle#L88

Including them like this is not how react-native-firebase works

Reply:
Yes I have added another PR adding just the above line that needs to be included into the build.gradle. This is not automatically handled by react-native.